app/scripts/directives/bindService.js

```diff
@@ -71,6 +71,7 @@ function BindService($scope,
                                         .concat(replicaSets)
                                         .concat(statefulSets);
       ctrl.applications = _.sortByAll(apiObjects, ['metadata.name', 'kind']);
```

Comment:
Curious if that is something that can be clarified.  "false" meaning "yes, but no" and none meaning "false" (ie, what you thought false would have meant) seems confusing.